pymongo/mongo_client.py

```diff
@@ -503,6 +503,11 @@ def __init__(
 
         .. seealso:: The MongoDB documentation on `connections <https://dochub.mongodb.org/core/connections>`_.
 
+        .. versionchanged:: 4.0
```

Comment:
Please combine this with the existing versionchanged below.

Reply:
Done.